data-types/strings/tests/test_strings.py

```diff
@@ -7,200 +7,208 @@ def test_one(self):
         greeting = "Hello!!\n"
         # In place of the line below, call a method to achieve the expected output.
-        actual = None
+        actual = greeting[:-1]
         expected = "Hello!!"
 
         assert actual == expected
 
-    @pytest.mark.skip
+
     def test_ten(self):
         greeting = "Hello!!\n\n"
         # In place of the line below, call a method to achieve the expected output.
-        actual = None
+        actual = greeting[:-1]
         expected = "Hello!!\n"
 
         assert actual == expected
 
-    @pytest.mark.skip
+
     def test_eleven(self):
         rhyme = "eeny, meeny, miny, moe"
         # In place of the line below, call a method to achieve the expected output.
-        actual = None
+        actual = ""
+        for character in rhyme:
+            if character != "e":
+                actual += character
         expected = "ny, mny, miny, mo"
 
         assert actual == expected
 
-    @pytest.mark.skip
     def test_twelve(self):
         rhyme = "eeny, meeny, miny, moe"
         # In place of the line below, call a method to achieve the expected output.
-        actual = None
+
+        actual = ""
+        for character in rhyme:
+            if character not in ["e", "i", "o"]:
+                actual += character
         expected = "ny, mny, mny, m"
 
         assert actual == expected
 
-    @pytest.mark.skip
     def test_thirteen(self):
         greeting = "Hello World!"
         # In place of the line below, call a method to get the number of characters in the string
-        actual = None
+        actual = len(greeting)
         expected = 12
 
         assert actual == expected
 
-    @pytest.mark.skip
+
     def test_fourteen(self):
         greeting = "Hello World!\n"
         # In place of the line below, call a method to get the number of characters in the string
-        actual = None
+        actual = len(greeting)
         expected = 13
 
         assert actual == expected
 
-    @pytest.mark.skip
     def test_fifteen(self):
         greeting = "Hello       World!"
         # In place of the line below, call a method to get the number of characters in the string
-        actual = None
+        actual = len(greeting)
         expected = 18
 
         assert actual == expected
 
-    @pytest.mark.skip
     def test_sixteen(self):
         greeting = "Hello World!"
         # In place of the line below, call a method to get the number of 'o' in the string
-        actual = None
+        actual = greeting.count("o")
         expected = 2
 
         assert actual == expected
 
-    @pytest.mark.skip
     def test_seventeen(self):
         greeting = "Hello World!"
         # In place of the line below, call a method to get the number of vowels in the string
-        actual = None
+        actual = 0
+
+        for i in greeting:
+            if (i == 'a' or i == 'e' or i == 'i' or i == 'o' or i == 'u' or i == 'A' or i == 'E' or i == 'I' or i == 'O' or i == 'U'):
```

Comment:
There is a cleaner way to check if `i` is one of these characters. Check one of your earlier solutions (hint: you can use the keyword `in`)